### ISSUE TITLE
--preloader option is not available

### ISSUE THREAD

**Issue by mouzei** [bkerler/mtkclient#1072]
Latest version.

python\python mtkclient\mtk.py printgpt --preloader preloader_bbk6750_66_l.bin
MTK Flash/Exploit Client Public V2.0.1 (c) B.Kerler 2018-2024

Traceback (most recent call last):
  File "xxx\mtkclient\mtk.py", line 980, in <module>
    mtk = Main(args).run(parser)
  File "xxx\mtkclient\mtkclient\Library\mtk_main.py", line 352, in run
    ArgHandler(self.args, config)
  File "xxx\mtkclient\mtkclient\Library\mtk_main.py", line 104, in __init__
    self.info("O:Var1:\t\t" + args.var1)
TypeError: can only concatenate str (not "NoneType") to str

**Comment by bkerler** [comment]
Fixed in [36eec49](https://github.com/bkerler/mtkclient/commit/36eec4904523702a78d01c6c6f1dfe97e095ac0a)

**Comment by mouzei** [comment]
> Fixed in [36eec49](https://github.com/bkerler/mtkclient/commit/36eec4904523702a78d01c6c6f1dfe97e095ac0a)

C:\Users\Administrator\Desktop\xxx>python\python mtkclient\mtk.py printgpt --preloader preloader_bbk6750_66_l.bin
MTK Flash/Exploit Client Public V2.0.1 (c) B.Kerler 2018-2024

Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\xxx\mtkclient\mtk.py", line 980, in <module>
    mtk = Main(args).run(parser)
  File "C:\Users\Administrator\Desktop\xxx\mtkclient\mtkclient\Library\mtk_main.py", line 352, in run
    ArgHandler(self.args, config)
  File "C:\Users\Administrator\Desktop\xxx\mtkclient\mtkclient\Library\mtk_main.py", line 104, in __init__
    self.info("O:Var1:\t\t" + config.chipconfig.var1)
TypeError: can only concatenate str (not "int") to str

**Comment by kngharv** [comment]
modified the line and still got the same error as well:

```
$ ./mtk.py plstage --preloader=preloader_g83v71c2k_dfl_tee.bin
MTK Flash/Exploit Client Public V2.0.1 (c) B.Kerler 2018-2024

Traceback (most recent call last):
  File "/home/kngharv/git/mtkclient-2.0.1/./mtk.py", line 980, in <module>
    mtk = Main(args).run(parser)
  File "/home/kngharv/git/mtkclient-2.0.1/mtkclient/Library/mtk_main.py", line 353, in run
    ArgHandler(self.args, config)
  File "/home/kngharv/git/mtkclient-2.0.1/mtkclient/Library/mtk_main.py", line 104, in __init__
    self.info("O:Var1:\t\t" + config.chipconfig.var1)
TypeError: can only concatenate str (not "int") to str
```
